src/software/ai/navigator/path_planner/path_planner_test.cpp

```diff
@@ -16,18 +16,16 @@
 // length at which to evaluate the spline value and check
 // if it intersects an obstacle
 static constexpr double PATH_CHECK_INTERVAL_M = 0.05;
-// distance to destination before the path is considered to
-// have reached the destination
+// distance to endination before the path is considered to
```

Comment:
What's `endination`? Looks like it replaced `destination` in a few places, maybe a bad find and replace?

Reply:
lmao you did a find and replace on `dest -> end` didn't you :eyes:

Reply:
yes you got me